components/molecule/phoneInput/src/index.js

```diff
@@ -102,28 +104,28 @@ export default function MoleculePhoneInput({
               src={flagIcons[selectedPrefix?.value]}
             />
           )}
-          {visiblePrefixes && (
-            <AtomIcon
-              size={ATOM_ICON_SIZES.medium}
-              color={ATOM_ICON_COLORS.currentColor}
-            >
-              {showDropdown ? dropdownCloseIcon : dropdownIcon}
-            </AtomIcon>
-          )}
+          {visiblePrefixes && (showDropdown ? dropdownCloseIcon : dropdownIcon)}
```

Comment:
```suggestion
          {visiblePrefixes && <PrimitiveInjector size={ATOM_ICON_SIZES.medium}
              color={ATOM_ICON_COLORS.currentColor}>(showDropdown ? dropdownCloseIcon : dropdownIcon)</PrimitiveInjector>}
```

Reply:
I think the problem is that the icons already come with a wrapper that includes that configuration, and it was being duplicated and overwritten. 
@Alverd04 could you confirm this? 🙇‍♀️